testing/lit.cfg

```diff
@@ -0,0 +1,8 @@
+# -*- Python -*-
+
+import lit.formats
+
+config.name = 'Dimsum'
+config.test_format = lit.formats.ShTest()
+config.substitutions.append(('%clang', 'clang -I ../../..'))
```

Comment:
clang++?

Reply:
`clang` also works. Nevertheless, changed to clang++